base/tps/src/main/java/org/dogtagpki/server/tps/processor/TPSProcessor.java

```diff
@@ -2387,6 +2387,7 @@ need to reach out to the Registration DB (authid)
             try {
                 tps.tdb.tdbAddTokenEntry(tokenRecord, TokenStatus.UNFORMATTED);
                 tps.tdb.tdbActivity(ActivityDatabase.OP_ADD, tokenRecord, session.getIpAddress(), logMsg, "success");
+                fillTokenRecordDefaultPolicy(tokenRecord);
```

Comment:
@jmagne Is this needed to fix this bug? If not maybe we should put it in a separate commit to avoid confusions (assuming it's needed for the `master` branch).

Reply:
edewata, under the crush we just put that tps related fix in this blob. I think it should be fine for master. This has to do with tps pin reset.

Reply:
Also edewata, this is just a missing piece of another commit that covers this pin reset issue. So that issue is represented already.

Reply:
Thanks for the clarification! This is fine.